source/common/upstream/health_checker_base_impl.cc

```diff
@@ -78,33 +78,19 @@ HealthCheckerImplBase::~HealthCheckerImplBase() {
   }
 }
 
-void HealthCheckerImplBase::decHealthy() {
-  ASSERT(local_process_healthy_ > 0);
```

Comment:
I see now that these stats are NeverImport, so is this just a cleanup since it basically does nothing? I think we can kill the member var also?

Reply:
oh I meant to back this out and do it as a separate PR.